cpp/src/arrow/parquet/writer.h

```diff
@@ -52,6 +54,14 @@ class FileWriter {
   std::unique_ptr<Impl> impl_;
 };
 
+/**
+ * Write a flat Table to Parquet.
+ *
+ * The table shall only consist of nullable, non-repeated columns of primitive type.
```

Comment:
No, that sentence should mean that REQUIRED and OPTIONAL but not REPEATED is supported.